<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-0PBS36T" name="aaa" tests="1" failures="1" timestamp="29 Dec 2020 07:55:23 GMT" time="29.586" errors="0">
  <testcase name="loginLogout" time="29.538" classname="tests.LoginTests">
    <failure type="java.lang.AssertionError" message="expected [https://opensource-demo.orangehrmlive.com/index.php/dashboard] but found [https://opensource-demo.orangehrmlive.com/index.php/auth/validateCredentials]">
      <![CDATA[java.lang.AssertionError: expected [https://opensource-demo.orangehrmlive.com/index.php/dashboard] but found [https://opensource-demo.orangehrmlive.com/index.php/auth/validateCredentials]
	at tests.LoginTests.loginLogout(LoginTests.java:70)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginLogout -->
</testsuite> <!-- aaa -->
